From f1bf907cc14f55edebac9d4633a4c1edec0f9f53 Mon Sep 17 00:00:00 2001
From: S. Lockwood-Childs <sjl@vctlabs.com>
Date: Fri, 4 Mar 2022 12:53:16 -0800
Subject: [PATCH] Remove hard dependency on nose

nose is not really a runtime dependency, but rather just a dependency
for running the unit tests.

Signed-off-by: S. Lockwood-Childs <sjl@vctlabs.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 0c733fe..e7533b8 100755
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,6 @@ setup(
     install_requires=[
     ],
     setup_requires=[
-        "nose>=1.3.7",
     ],
     dependency_links=[
     ],
@@ -31,6 +30,7 @@ setup(
     },
     tests_require=[
         "coverage>=3.5.2",
+        "nose>=1.3.7",
         "PyHamcrest>=1.9.0",
     ],
     classifiers=[
-- 
2.20.1

